style.css

```diff
@@ -0,0 +1,665 @@
+  width: 122px;
+}
+
+.languages-elements {
+  display: flex;
+  flex-direction: row;
+  font-family: "Poppins", sans-serif;
+  gap: 10px;
+  padding-inline-start: 0;
+  padding: 12px 15px;
+}
+
+.Language-Logo {
+  width: 70px;
+}
+
+.javaScript-logo {
+  border-radius: 90px;
+}
+
+.languages-elements li p {
+  color: #253858;
+  font-size: 1.5rem;
+  font-weight: 500;
+}
+
+.contact-MeForm {
+  background-color: hsl(0, 0%, 90%);
+  margin-top: 100px;
+}
+
+.contact-me {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  text-align: center;
+  font-weight: 700;
+  font-size: 40px;
+  padding: 23px;
+  color: hsl(0, 0%, 100%);
+}
+
+.form-wrapper {
+  max-width: 600px;
+  padding: 20px;
+}
+
+.ContactDesctiption {
+  margin-bottom: 20px;
+  font-family: "Poppins", sans-serif;
+  font-size: 17px;
+  text-align: center;
+  color: hsl(240, 100%, 96%);
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  font-family: 'Poppins', sans-serif;
+}
+
+.form-group {
+  margin-bottom: 20px;
+}
+
+input,
+textarea {
+  width: 100%;
+  padding: 10px;
+  border: 1px solid #ccc;
+  font-family: 'Poppins', sans-serif;
+  border-radius: 4px;
+}
+
+textarea {
+  height: 150px;
+}
+
+.btntouch {
+  margin-bottom: 20px;
+  text-align: center;
+}
+
+.continer {
+  text-align: center;
+}
+
+.btnGettouch {
+  padding: 10px 15px;
+  border-radius: 8px;
+  color: hsla(234, 100%, 69%, 1);
+  background-color: hsl(0, 0%, 100%);
+  border: 1px solid #396df2;
+  width: fit-content;
+  cursor: pointer;
+  font-size: 17px;
+  font-weight: 600;
+}
+
+.btncontainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.btnGettouch:hover {
+  background-color: hsl(235, 30%, 93%);
+}
+
+.card-box {
+  width: 50%;
+  flex: 1;
+  box-sizing: border-box;
+  float: left;
+  padding: 4px;
+}
+
+.about_myself__menu {
+  width: 40%;
+  flex: 1;
+  box-sizing: border-box;
+  float: left;
+  padding: 4px;
+}
+
+.about-myself__block {
+  width: 60%;
+  flex: 1;
+  box-sizing: border-box;
+  float: left;
+  padding: 4px;
+}
+
+.contact-application::after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+/* menu */
+.menu-lists {
+  list-style: none;
+}
+
+.menu-mobile {
+  display: none;
+}
+
+@media (max-width: 768px) {
+  .menu-lists {
+    display: none;
+  }
+
+  .menu-lists.active {
+    display: block;
+    margin-top: 10px;
+  }
+}
+
+@media (min-width: 769px) {
+  .info-image {
+    display: none;
+  }
+
+  .menu-lists {
+    display: flex;
+    margin-left: 20px;
+  }
+
+  .menu-lists li {
+    margin-left: 15px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: hsl(218, 31%, 30%);
+  }
+}
+
+.menu-mobile.active {
+  display: block;
+}
+
+@media (min-width: 768px) {
+  .menu-mobile {
+    display: none;
+  }
+}
+
+@media (max-width: 768px) {
+  .desktop-menu-container {
+    display: none;
+  }
```

Comment:
- [ ] Once again as mentioned by the previous reviewer you should only use 1 media query as an adaptation for the desktop and it should be similar to this: 👇
    ```
    @media screen and (min-width: 1080px) { 
      // your dektop stylings go here...
    }
    ```